src/styles/blocks/card.scss

```
@@ -0,0 +1,138 @@
+.card {
+  box-sizing: border-box;
+
+  width: $catalog-card-width;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  transition-duration: $transition-time;
+
+  &__image-container {
+    margin: 32px 19px 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__caption {
+    margin: 36px 16px 0;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: start;
+  }
+
+  &__title {
+    margin: 0;
+    padding: 0;
+
+    text-align: left;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+
+    text-align: left;
+    color: $color-secondary;
+  }
+
+  &__reviews {
+    margin: 16px 16px 0;
+    padding: 0;
+
+    width: 166px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews-number {
+    margin: 0;
+    padding: 0;
+
+    justify-content: end;
+    align-items: center;
+  }
+
+  &__price {
+    margin: 24px 16px 0;
+
+    width: 166px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-label {
+    margin: 0;
+    padding: 0;
+
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+
+    color: $color-secondary;
+  }
+
+  &__price-value {
+    margin: 0;
+    padding: 0;
+
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+  }
+
+  &__buy-link {
+    box-sizing: border-box;
+
+    margin: 16px;
+
+    width: 166px;
+    height: 40px;
+
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    background-color: $color-blue-accent;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    color: #fff;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__buy-link:hover {
```

You can move it inside the `&__buy-link` and use this syntax
```suggestion
  &:hover {
```